league_rpc_linux/__main__.py

```diff
@@ -35,6 +43,10 @@ def main(cli_args: argparse.Namespace):
         client_id=cli_args.client_id,
         wait_for_discord=cli_args.wait_for_discord,
     )
+
+    p = Process(target=LCU_Thread.start_connector, args=(rpc,))
```

Comment:
Could you please explain what this will do exactly? and why args=(rpc,) is needed.

Maybe placing a comment above, giving a short summary of why this is needed would be good for future reference.

Reply:
```python
    # Start LCU_Thread
    # This process will connect to the LCU API and updates the rpc based on data subscribed from the LCU API.
    # In this case passing the rpc object to the process is easier than trying to return updated data from the process.
    # Every In-Client update will be handled by the LCU_Thread process and will update the rpc accordingly.
```